runtime/compiler/env/J9ClassEnv.cpp

```diff
@@ -78,9 +116,49 @@ J9::ClassEnv::classInstanceSize(TR_OpaqueClassBlock * clazzPointer)
 J9ROMClass *
 J9::ClassEnv::romClassOf(TR_OpaqueClassBlock * clazz)
    {
-   return TR::Compiler->cls.convertClassOffsetToClassPtr(clazz)->romClass;
+   J9Class *j9clazz = TR::Compiler->cls.convertClassOffsetToClassPtr(clazz);
+   return j9clazz->romClass;
    }
 
+J9Class **
+J9::ClassEnv::superClassesOf(TR_OpaqueClassBlock * clazz)
+   {
+   return TR::Compiler->cls.convertClassOffsetToClassPtr(clazz)->superclasses;
+   }
+
+J9ROMClass *
+J9::ClassEnv::romClassOfSuperClass(TR_OpaqueClassBlock * clazz, size_t index)
+   {
+   return self()->superClassesOf(clazz)[index]->romClass;
+   }
+
+J9ITable *
+J9::ClassEnv::iTableOf(TR_OpaqueClassBlock * clazz)
+   {
+   return (J9ITable*) self()->convertClassOffsetToClassPtr(clazz)->iTable;
+   }
+
+J9ITable *
+J9::ClassEnv::iTableNext(J9ITable *current)
+   {
+   return current->next;
+   }
+
+J9ROMClass *
+J9::ClassEnv::iTableRomClass(J9ITable *current)
+   {
+   return current->interfaceClass->romClass;
+   }
+
+std::vector<TR_OpaqueClassBlock *>
+J9::ClassEnv::getITable(TR_OpaqueClassBlock *clazz)
```

Comment:
This must be one the queries used just JITServer. Maybe it needs #ifdef JITSERVER_SUPPORT

Reply:
In this case `ClassEnv::getITable` will be removed from this PR since this one takes care of the common methods between the `master` and `JITServer`. `ClassEnv::getITable` will be merged by #7016.